### ISSUE TITLE
JSON formatting - camelcase?

### ISSUE THREAD

**Issue by kurtseifried** [cloudsecurityalliance/gsd-project#29]
Should we officially use camel case? Google guidelines? https://google.github.io/styleguide/jsoncstyleguide.xml 

Whatever we do, consistency is key.

**Comment by athix** [comment]
I say we go with Google's style guide for now, and if for some reason there's a disagreement with the guide we can revisit the discussion then. It will provide a clear answer for how to style things, and we can potentially validate against it for consistency.

That said, the only thing that's important to me is to be consistent, and to as much as possible follow industry best practices on formatting.

**Comment by joshbuker** [comment]
I agree that the important thing is consistency. I'd be fine using either snake_case or camelCase, as I use both frequently between Ruby and JavaScript respectively.

With the style guide, if there's a community sourced guide similar to the [Ruby Style Guide](https://rubystyle.guide), that would be preferable. I don't know of any currently, and the Google style guide was handily available (and I think is what OSV follows).
